aiida/cmdline/commands/cmd_database.py

```diff
@@ -8,95 +8,61 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 """`verdi database` commands."""
+# pylint: disable=unused-argument
 
 import click
 
 from aiida.backends.general.migrations.duplicate_uuids import TABLES_UUID_DEDUPLICATION
 from aiida.cmdline.commands.cmd_verdi import verdi
 from aiida.cmdline.params import options
 from aiida.cmdline.utils import decorators, echo
-from aiida.common import exceptions
 
 
 @verdi.group('database')
 def verdi_database():
-    """Inspect and manage the database."""
+    """Inspect and manage the database.
+
+    .. deprecated:: v3.0.0
+    """
 
 
 @verdi_database.command('version')
 def database_version():
     """Show the version of the database.
 
     The database version is defined by the tuple of the schema generation and schema revision.
-    """
-    from aiida.manage.manager import get_manager
 
-    manager = get_manager()
-    manager._load_backend(schema_check=False)  # pylint: disable=protected-access
-    backend_manager = manager.get_backend_manager()
-
-    echo.echo('Generation: ', bold=True, nl=False)
-    echo.echo(backend_manager.get_schema_generation_database())
-    echo.echo('Revision:   ', bold=True, nl=False)
-    echo.echo(backend_manager.get_schema_version_database())
+    .. deprecated:: v2.1.0
+    """
+    echo.echo_warning(
+        'This command has been deprecated and will be removed soon. '
+        'The same information should now be available in `verdi status`.\n'
```

Comment:
Wouldn't use the word "should". Either it is or it isn't.

Reply:
Done.